### DISCUSSION TITLE
Helm chart deployment

### DISCUSSION THREAD

**Discussion by sddan** [firefly-iii/firefly-iii#4705] [General]
Hi, I've created a Firefly-III Helm chart for easier deployment and upgrading on Kubernetes. At the moment, I'm using it privately, but I would like to contribute it to the project in case anyone else has the same use case. How would I go about doing that?

**Comment by JC5** [comment]
Another user has been working on this as well. Your code could become part of the repository. Would you like that? See also: https://github.com/firefly-iii/firefly-iii/issues/4266

**Reply by sddan** [reply]
Ah, my bad. I didn't see that. Sure, will look into that. Thanks
